src/libs/PolicyUtils.js

```diff
@@ -7,22 +7,28 @@ import ONYXKEYS from '../ONYXKEYS';
  * Checks if we have any errors stored within the POLICY_MEMBER_LIST. Determines whether we should show a red brick road error or not.
  * Data structure: {email: {role:'user', errors: []}, email2: {role:'admin', errors: [{1231312313: 'Unable to do X'}]}, ...}
  *
- * @param {Object} policyMemberList
+ * @param {Object} [policyMemberList]
  * @returns {Boolean}
  */
 function hasPolicyMemberError(policyMemberList) {
+    if (!policyMemberList) {
```

Comment:
I don't think this is necessary since it works OK to pass `undefined` to `_.some()`

<img width="415" alt="image" src="https://user-images.githubusercontent.com/1228807/187225272-8992e7cf-f3bf-42b4-b549-f19c643b025e.png">